packages/core/src/Stores/traders-hub-store.js

```diff
@@ -519,17 +503,10 @@ export default class TradersHubStore extends BaseStore {
         );
     }
     getAvailableCTraderAccounts() {
-        if (this.CFDs_restricted_countries || this.financial_restricted_countries) {
-            this.available_ctrader_accounts = [];
-            return;
-        }
-
-        if (this.is_eu_user && !this.is_demo_low_risk) {
-            this.available_ctrader_accounts = this.available_cfd_accounts.filter(
-                account =>
-                    ['EU', 'All'].some(region => region === account.availability) &&
-                    account.platform === CFD_PLATFORMS.CTRADER
-            );
+        if (this.root_store.client.is_logged_in) {
```

Comment:
You're just checking the availability when user is logged in, we need to make this dynamic for the logged out dashboard as well.